### PR TITLE
Use Python 3 in Quick Start example

### PR DESCRIPTION
**Pull request by stevepiercy**
Subject: Use Python 3 in Quick Start example

### Feature or Bugfix
- Feature

### Purpose
- Python 2 has been EOLed, and we should use Python 3 going forward.

### CODE REVIEW

File: sphinx/templates/quickstart/conf.py_t [hunk 1]
```diff
@@ -105,7 +105,7 @@ html_static_path = ['{{ dot }}static']
 # -- Options for intersphinx extension ---------------------------------------
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'https://docs.python.org/': None}
+intersphinx_mapping = {'https://docs.python.org/3/': None}
```
Comment (tk0miya) [inline sphinx/templates/quickstart/conf.py_t:108]: Why is this needed? If my understanding correct, `https://docs.python.org/` returns appreciate URL of python document. It is not only for 2.x.
```
$ curl -I https://docs.python.org/
HTTP/2 302
server: nginx
content-type: text/html
location: https://docs.python.org/3/
x-clacks-overhead: GNU Terry Pratchett
strict-transport-security: max-age=315360000; includeSubDomains; preload
via: 1.1 varnish
accept-ranges: bytes
date: Thu, 01 Aug 2019 15:47:56 GMT
via: 1.1 varnish
age: 112150
x-served-by: cache-jfk8140-JFK, cache-tyo19950-TYO
x-cache: HIT, HIT
x-cache-hits: 1, 3
x-timer: S1564674477.715171,VS0,VE0
content-length: 154
```

Comment (stevepiercy) [reply sphinx/templates/quickstart/conf.py_t:108]: It saves the network overhead of a second client request from the redirect: `HTTP/2 302`.

😆 : `x-clacks-overhead: GNU Terry Pratchett` oh, Python peeps are the best!

Comment (tk0miya) [reply sphinx/templates/quickstart/conf.py_t:108]: Hmm... The title and purpose of this PR are not related with change, right? If true, I can understand what are you doing. Indeed, this reduces overhead of communication.

**Review by tk0miya — CHANGES_REQUESTED** [review]
Could you rebase this into 2.0 branch? Then I'll merge this soon.

**Comment by stevepiercy** [comment]
Bugger. Closing in favor of a proper rebase in #6626. Also found a second link that was out of date and now uses https.
